src/libs/actions/Policy.js

```diff
@@ -112,6 +112,15 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Does the user have any (free policy) workspaces?
+ *
+ * @returns {Boolean}
+ */
+function hasWorkspaces() {
```

Comment:
is it possible to have workspaces that are not free? If it is possible, maybe this function should be renamed to `hasFreeWorkspaces`.

Reply:
Great question! I don't think so, not at this point at least 👍